---
title: 例外処理ステートメント - C# リファレンス
ms.custom: seodec18
ms.date: 07/20/2015
helpviewer_keywords:
- keywords [C#], exception handling
ms.assetid: 6d0323a3-3164-411c-9b84-a3606bd0e13c
ms.openlocfilehash: 835bc0a0ac8bdabccddc12f0b7d3eaec4ecd3dee
ms.sourcegitcommit: bdd930b5df20a45c29483d905526a2a3e4d17c5b
ms.translationtype: HT
ms.contentlocale: ja-JP
ms.lasthandoff: 12/11/2018
ms.locfileid: "53243154"
---
# <a name="exception-handling-statements-c-reference"></a><span data-ttu-id="65a5a-102">例外処理ステートメント (C# リファレンス)</span><span class="sxs-lookup"><span data-stu-id="65a5a-102">Exception Handling Statements (C# Reference)</span></span>
<span data-ttu-id="65a5a-103">C# では、プログラムの実行時に発生する例外と呼ばれる異常事態を処理する組み込みサポートが用意されています。</span><span class="sxs-lookup"><span data-stu-id="65a5a-103">C# provides built-in support for handling anomalous situations, known as exceptions, which may occur during the execution of your program.</span></span> <span data-ttu-id="65a5a-104">このような例外は、通常の制御フローの外部にあるコードによって処理されます。</span><span class="sxs-lookup"><span data-stu-id="65a5a-104">These exceptions are handled by code that is outside the normal flow of control.</span></span>  
  
 <span data-ttu-id="65a5a-105">ここでは、例外処理に関する次のトピックについて説明します。</span><span class="sxs-lookup"><span data-stu-id="65a5a-105">The following exception handling topics are explained in this section:</span></span>  
  
-   [<span data-ttu-id="65a5a-106">throw</span><span class="sxs-lookup"><span data-stu-id="65a5a-106">throw</span></span>](../../../csharp/language-reference/keywords/throw.md)  
  
-   [<span data-ttu-id="65a5a-107">try-catch</span><span class="sxs-lookup"><span data-stu-id="65a5a-107">try-catch</span></span>](../../../csharp/language-reference/keywords/try-catch.md)  
  
-   [<span data-ttu-id="65a5a-108">try-finally</span><span class="sxs-lookup"><span data-stu-id="65a5a-108">try-finally</span></span>](../../../csharp/language-reference/keywords/try-finally.md)  
  
-   [<span data-ttu-id="65a5a-109">try-catch-finally</span><span class="sxs-lookup"><span data-stu-id="65a5a-109">try-catch-finally</span></span>](../../../csharp/language-reference/keywords/try-catch-finally.md)  
  
## <a name="see-also"></a><span data-ttu-id="65a5a-110">参照</span><span class="sxs-lookup"><span data-stu-id="65a5a-110">See Also</span></span>  

- [<span data-ttu-id="65a5a-111">C# リファレンス</span><span class="sxs-lookup"><span data-stu-id="65a5a-111">C# Reference</span></span>](../../../csharp/language-reference/index.md)  
- [<span data-ttu-id="65a5a-112">C# プログラミング ガイド</span><span class="sxs-lookup"><span data-stu-id="65a5a-112">C# Programming Guide</span></span>](../../../csharp/programming-guide/index.md)  
- [<span data-ttu-id="65a5a-113">C# のキーワード</span><span class="sxs-lookup"><span data-stu-id="65a5a-113">C# Keywords</span></span>](../../../csharp/language-reference/keywords/index.md)  
- [<span data-ttu-id="65a5a-114">ステートメントのキーワード</span><span class="sxs-lookup"><span data-stu-id="65a5a-114">Statement Keywords</span></span>](../../../csharp/language-reference/keywords/statement-keywords.md)  
- [<span data-ttu-id="65a5a-115">例外と例外処理</span><span class="sxs-lookup"><span data-stu-id="65a5a-115">Exceptions and Exception Handling</span></span>](../../../csharp/programming-guide/exceptions/index.md)
